examples/webrx/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
by the way this is not true. We need all front-end deps in the repo, could you add them? Similar to the other examples. It's needed for the site to run fine on the live site.